src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/InternalCalls.cs

```diff
@@ -41,7 +41,7 @@ internal enum ClassLibFunctionId
         // UnhandledExceptionHandler = 2, // unused
         AppendExceptionStackFrame = 3,
         // unused = 4,
-        GetSystemArrayEEType = 5,
+        // GetSystemArrayEEType = 5, // unused
```

Comment:
We do not need to keep space for these unused values around. This can be deleted and the rest packed (you can also delete the existing unused values).